src/StanfordBehat/DrupalExtension/Context/SwsMinkContext.php

```diff
@@ -30,6 +30,31 @@ public function beforeScenario(BeforeScenarioScope $scope) {
     $this->currentFeature = $scope->getFeature();
   }
 
+  /**
+   * Set the window to a breakpoint size.
+   *
+   * @param string $size
+   *   Small, Medium, Large, or 'Extra Large' screen.
+   *
+   * @Then I set the window size to :size
+   */
+  public function iSetWindowSize($size) {
+    $sizes = [
+      'small' => [576, 320],
+      'medium' => [768, 1024],
```

Comment:
Are these `max`, `min`?
```suggestion
      'medium' => [1024, 768],
```

Reply:
window width, height values. i just chose some height that would simulate a portrait tablet.